### PR TITLE
add the control files for the travis CI system

### PR DESCRIPTION
**Pull request by andlaus**
this patch is based on the travis files from opm-material, so it may be that they work. (I've no experience with travis yet, nor do I know a way to test this besides the live system, so bear with me.)

this will not break anything outside travis, so I'll be self-merging immediately.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - liblapack-dev
+      - libgmp3-dev
+      - libsuitesparse-dev
+      - libeigen3-dev
+
+before_script:
+    - export CXX="g++-4.8" CC="gcc-4.8" FC="gfortran-4.8"
+    - cd ..
+    - git clone https://github.com/OPM/opm-common.git
+    - opm-common/travis/clone-opm.sh opm-material
```
Comment (joakim-hove) [inline .travis.yml]: This should be: `opm-common/travis/clone-opm.sh ewoms`

Comment (andlaus) [inline .travis.yml]: thanks for spotting this, i've pushed a fix (without a PR)

**Comment by joakim-hove** [comment]
Observe that the general build script is `opm-common/travis/build-and-test.sh` - i.e. the `build-and-test-ewoms.sh` and `build-ewoms.sh` scripts will not be used[1].

[1]: Yes - there are some dangling files of that type in some of the other repositories, which should be removed.

**Comment by andlaus** [comment]
> Observe that the general build script is opm-common/travis/build-and-test.sh - i.e. the build-and-test-ewoms.sh and build-ewoms.sh scripts will not be used[1].

I was wondering about that a bit while doing this. I'll remove these files, thanks for the hint!

**Pull request merged**